src/uwtools/api/ungrib.py

```diff
@@ -0,0 +1,53 @@
+"""
+API access to the uwtools ungrib driver.
```

Comment:
```suggestion
API access to the ``uwtools`` ``ungrib`` driver.
```
Looks nice when RST renders to HTML.